### PR TITLE
Change recording setting position

### PR DESCRIPTION
**Pull request by affan00**
@azharmalik3

### CODE REVIEW

File: app/assets/javascripts/cameras/single/cloud_recording_schedule.js.coffee [hunk 1]
```diff
@@ -233,54 +256,74 @@ handleStatusSelect = ->
...
-        hideEditButton()
         hideFrequencySelect()
         hideDurationSelect()
+        hideScheduleCalendar()
         updateScheduleToOff()
-        showScheduleLabel()
 
 renderCloudRecordingDuration = ->
   $("#cloud-recording-duration").val(Evercam.Camera.cloud_recording.storage_duration)
+  switch Evercam.Camera.cloud_recording.storage_duration
```
Comment (azharmalik3) [inline app/assets/javascripts/cameras/single/cloud_recording_schedule.js.coffee]: Don't use switch just get text from dropdown.

File: app/assets/javascripts/cameras/single/snapshots_navigator.js.coffee [hunk 2]
```diff
@@ -973,12 +959,32 @@ recodringSnapshotDivHeight = ->
...
     return
 
+onCollapsRecording = ->
+  $('#cloud-recording-collaps').click ->
+    $('#cloud-recording-calendar').toggleClass 'open'
+
+checkCalendarDisplay = ->
+  if $('.col-recording-right').css('display') == 'none'
+    $('#recording-tab .left-column').animate { width: "99.8%" }, ->
+      recodringSnapshotDivHeight()
```
Comment (azharmalik3) [inline app/assets/javascripts/cameras/single/snapshots_navigator.js.coffee]: Use like this:
`$('#recording-tab .left-column').animate { width: "99.8%" }, recodringSnapshotDivHeight`

File: app/assets/stylesheets/views/cameras/single/_snapshot-recorder.scss [hunk 3]
```diff
@@ -66,10 +67,18 @@
       width: 25px;
       height: 25px;
     }
+
+    .setting-heading .btn {
```
Comment (affan00) [inline app/assets/stylesheets/views/cameras/single/_snapshot-recorder.scss]: It already has minimum depth if we reduce the depth it doesn't show the desired result.

**Comment by azharmalik3** [comment]
:+1:

**Pull request merged**